### ISSUE TITLE
Paired data with more than 2 measures don't show slope plot

### ISSUE THREAD

**Issue by markwhiting** [ACCLAB/dabestr#72]
## Problem
Paired measures with more than 2 measures don't show a `slopegraph`, meaning that the main data points are not visually paired. 

![image](https://user-images.githubusercontent.com/673171/79073775-5fb7e200-7cb6-11ea-9e85-e210b3fdcd62.png)
One fix is to use color, e.g., 
```R
color.column = groups
```
![image](https://user-images.githubusercontent.com/673171/79073789-78c09300-7cb6-11ea-8279-ee45d61f9d9e.png)

But with large datasets this is hard to parse, and requires some theme adjustment to avoid a huge legend, e.g.,

```R
theme = ggplot2::theme_classic() + theme(legend.position = "none")
```

## Solution
Ideally, each group in each measure would be connected with a line to consecutive measures.

**Comment by josesho** [comment]
Hi @markwhiting , 

This feature (successive repeated measures) has been in the pipeline for a while! Whiel we cannot give a window for its eventual implementation, we are working on it!

Happy to receive a PR if you already have something hacked up.

Joses

**Comment by josesho** [comment]
@markwhiting , just FYI, this feature has also been [requested](https://github.com/ACCLAB/DABEST-python/issues/68) (and is in the pipeline) for the Python version of dabestr.

**Comment by josesho** [comment]
cf #108
